### DISCUSSION TITLE
nushell support

### DISCUSSION THREAD

**Discussion by gokulyc** [pypa/pipenv#5311] [Ideas]
Is there a support for nushell?

if no, Is there plan to support.

currently pipenv not working for me as expected in nushell

**Comment by matteius** [comment]
Never heard of it -- as far as I know, nobody is working on this yet.  Is this something you would be interested in working on supporting?

**Reply by gokulyc** [reply]
https://github.com/nushell/nushell

poetry has the support. I use nushell extensively in Windows because of better features than command prompt.

**Comment by oz123** [comment]
Looking at poetry all we need to do is bump shellingham and we are fine.

**Comment by oz123** [comment]
I created an issue for this https://github.com/pypa/pipenv/issues/5333

**Comment by oz123** [comment]
This issue is now fixed, as soon as we release a new version you can use nushell.
